base/loading.jl

```diff
@@ -104,11 +104,10 @@ function find_source_file(path::String)
     return isfile(base_path) ? base_path : nothing
 end
 
-function find_all_in_cache_path(mod::Symbol)
-    name = string(mod)
+function find_all_in_cache_path(name::String)
```

Comment:
This change doesn't consist with precompile.jl 
https://github.com/JuliaLang/julia/blob/324dc4942050d6663fe472a657c2f6f1baf2821e/base/precompile.jl#L1393